### ISSUE TITLE
Pweb error. Previous picture disappears in new version.

### ISSUE THREAD

**Issue by digipl** [maidsafe/safe_browser#1180]
1.-I create a web page with a picture (wave.jpg). The image is displayed correctly
2.-I upload a new version with a different picture but the same name (wave.jpg) and both versions show the last image. The first image has disappeared.

**Comment by joshuef** [comment]
thanks @digipl , will check this out

**Comment by joshuef** [comment]
If you have a repro site on the shared vault can you share that please 🙇

**Comment by digipl** [comment]
safe://digipl

There are 4 versions and each one has been edited with a different picture
but with the same name (wave.jpg).
In the four versions I only see the last picture, the three previous ones
have disappeared.

Saludos...

**Comment by joshuef** [comment]
what version of the browser are you using @digipl ? I'm seeing decode errors here and wondering if im running something not compatible with the upload.

**Comment by digipl** [comment]
0.15.04 linux. Is the version that open the Safe Network App.

I have the safe-browser-v0.16.0-alpha.24 too but I cannot access the sites
with this version.

**Comment by joshuef** [comment]
Right, aye, I think the older browser version had an issue w/ writing unencrypted data.

I'll try and replicate locally with an image of the same name 👍
